src/MultiCascader/utils.ts

```diff
@@ -183,7 +185,7 @@ export function useFlattenData(data: ItemDataType[], itemKeys: ItemKeys) {
   );
 
   useEffect(() => {
-    setFlattenData(flattenTree(data));
+    setFlattenData(flattenTree(data, itemKeys.childrenKey));
   }, [data]);
```

Comment:
ESLint reports that `itemKeys.childrenKey` is missing from `useEffect` dependencies.
```
React Hook useEffect has a missing dependency: 'itemKeys.childrenKey'. Either include it or remove the dependency array. If 'setFlattenData' needs the current value of 'itemKeys.childrenKey', you can also switch to useReducer instead of useState and read 'itemKeys.childrenKey' in the reducer
```